styles.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+.nav-list {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.brand-logo {
+  display: flex;
+  align-items: center;
+  font-weight: 700;
+  font-size: 25px;
+}
+
+.logo-link {
+  color: #6070ff;
+  font-size: 25px;
+  font-weight: 700;
+}
+
+.bi-list {
+  margin-top: px;
+  width: 30px;
+  height: 30px;
+  color: #6070ff;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 15px;
+  padding-right: 15px;
+  top: 0;
+  left: 0;
+  right: 0;
+  position: fixed;
+  z-index: 1000;
+}
+
+#intro {
+  background-image: url(/Assets/background-images/header-shapes\ mobile.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 80px;
+}
```

Comment:
I can see that the background image of this section is not visible. There might be an issue with the URL you provided for the background image. 😊
![broken](https://user-images.githubusercontent.com/33268927/170671560-23cd7524-2215-416f-80cf-9293befe6c59.png)

Reply:
I have rectified this